pkgs/build-support/bintools-wrapper/ld-wrapper.sh

```diff
@@ -47,6 +48,11 @@ if [[ "${NIX_ENFORCE_PURITY:-}" = 1 && -n "${NIX_STORE:-}"
             # Our ld is not built with sysroot support (Can we fix that?)
             :
         else
+            if [[ "$p" = -static || "$p" = -static-pie ]]; then
```

Comment:
There are a few other flags from the man page like `-Bstatic`, `-dn`, and `-non_shared` may be relevant (I'm not sure if any of them are used in practice though)

Reply:
Interesting. Do they have the same semantic.